Fix compile error when tracing enabled

The build fails when these options are enabled.
* CONFIG_IRQSOFF_TRACER
* CONFIG_PREEMPT_TRACER

Maybe it is not the cleanest solution, but it works...

Signed-off-by: Remy Bohmer <linux@bohmer.net>

---
 include/asm-generic/kdebug.h |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.26/include/asm-generic/kdebug.h
===================================================================
--- linux-2.6.26.orig/include/asm-generic/kdebug.h	2009-05-03 20:17:26.000000000 +0200
+++ linux-2.6.26/include/asm-generic/kdebug.h	2009-05-03 20:17:50.000000000 +0200
@@ -2,6 +2,7 @@
 #define _ASM_GENERIC_KDEBUG_H
 
 enum die_val {
+        DIE_OOPS = 1,
 	DIE_UNUSED,
 };
 
